<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVSimulationPerformanceOptions</class>
 <widget class="QWidget" name="SVSimulationPerformanceOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Time integration method</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="labelIntegrator">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Integrator:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxIntegrator"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMaxOrder">
        <property name="text">
         <string>Maximum Method Order (1-5):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditMaxOrder"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelNonLin">
        <property name="text">
         <string>Non-linear Iteration Convergence Coefficient:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditNonLin"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Linear equation system solver</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLesSolver">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Linear Equation System Solver:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxLesSolver"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMaxKry">
        <property name="text">
         <string>Maximum Dimension for Krylov Subspace:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditMaxKry"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelIterative">
        <property name="text">
         <string>Linear Iteration Convergence Coefficient:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditIterative"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Preconditioner for iterative equation system solvers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPreCond">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Preconditioner:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxPreCond"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPreILU">
        <property name="text">
         <string>Level of fill-in (ILU):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditPreILU"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>General options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0,0">
      <item row="0" column="0">
       <widget class="QLabel" name="labelInitialDT">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Initial time step:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QtExt::ParameterEdit" name="lineEditInitialDT"/>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBoxInitialDT"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMinDT">
        <property name="text">
         <string>Time step size limit (model consistency):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtExt::ParameterEdit" name="lineEditMinDT"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBoxMinDT"/>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelRelTol">
        <property name="text">
         <string>Relative Tolerance:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QtExt::ParameterEdit" name="lineEditRelTol"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;The options in this page only have an impact on simulation performance, not on results.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
   <slots>
    <signal>editingFinishedSuccessfully()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QtExt::ParameterEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ParameterEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxIntegrator</tabstop>
  <tabstop>lineEditMaxOrder</tabstop>
  <tabstop>lineEditNonLin</tabstop>
  <tabstop>comboBoxLesSolver</tabstop>
  <tabstop>lineEditMaxKry</tabstop>
  <tabstop>lineEditIterative</tabstop>
  <tabstop>comboBoxPreCond</tabstop>
  <tabstop>lineEditPreILU</tabstop>
  <tabstop>lineEditInitialDT</tabstop>
  <tabstop>comboBoxInitialDT</tabstop>
  <tabstop>lineEditMinDT</tabstop>
  <tabstop>comboBoxMinDT</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
